api/v1alpha1/contentsourcebinding_types.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ContentSourceReference contains info to locate a Kind ContentSource object
+type ContentSourceReference struct {
+	// API version of the referent.
+	APIVersion string `json:"apiVersion,omitempty"`
+	// Kind is the type of resource being referenced.
+	Kind string `json:"kind,omitempty"`
+	// Name is the name of resource being referenced.
+	Name string `json:"name"`
+}
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Namespaced,shortName=contentsourcebinding
```

Comment:
Having the `shortName` as the same name as the resource doesnt add a lot of value. Maybe we should name it to `csbinding` or even `csb`?  Historically, `shortname` is supposed to be really short (E.g. `ig`, `ma` etc. 

https://github.com/kubernetes-sigs/controller-tools/blob/master/pkg/crd/markers/crd.go#L258)

Reply:
I think we should omit the shortname, for now. We don't have it one the corresponding ContentSource.

Reply:
I have omitted the shortName in the latest diff. Thanks.